[Don't Merge] Jan 2026 release notes (#685)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/index.mdx

The following sections provide high-level summaries of each release suite and details about the compatibility between storefront components. To see more details about each drop-in component by version, go to the [changelog](/releases/changelog/).

## January 2026 suite

This suite release delivers several enhancements to the Drop-in SDK and drop-in components. The following sections provide details about the improvements. To see more details about each drop-in component by version, go to the [changelog](/releases/changelog/).

<Aside type="caution" title="Breaking changes">
This release introduces the following breaking changes:

> - **Major version 3.0.0 for B2C drop-ins**: All B2C drop-in components have been upgraded to version 3.0.0. These drop-ins have been tested on Adobe Commerce as a Cloud Service (ACCS) and Adobe Commerce 2.4.7+ for Platform-as-a-Service (PaaS) deployments. However, the revised license agreement restricts use to Adobe Commerce as a Cloud Service (ACCS) and Adobe Commerce Optimizer (ACO) only. Use on Platform-as-a-Service (PaaS) or on-premise deployments is not permitted under the license agreement.

Reviewer: ```suggestion
- **Major version 3.0.0 for B2C drop-ins**: All B2C drop-in components have been upgraded to version 3.0.0. These drop-ins have been tested on Adobe Commerce as a Cloud Service (SaaS) and Adobe Commerce 2.4.7+ for Platform-as-a-Service (PaaS) deployments. However, the revised license agreement restricts use to Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer only. Use on PaaS or on-premise deployments is not permitted under the license agreement.
```

> - **Version 1.0.0 for B2B drop-ins**: All B2B drop-in components are released at version 1.0.0. B2B drop-ins have been tested exclusively on Adobe Commerce as a Cloud Service (ACCS). However, the license agreement permits use on Adobe Commerce as a Cloud Service (ACCS) and Adobe Commerce Optimizer (ACO).

Reviewer: ```suggestion
- **Version 1.0.0 for B2B drop-ins**: All B2B drop-in components are released at version 1.0.0. B2B drop-ins have been tested exclusively on Adobe Commerce as a Cloud Service. However, the license agreement permits use on Adobe Commerce as a Cloud Service and Adobe Commerce Optimizer.
```

- **Checkout drop-in**: Billing address now uses `customerAddressUid` field instead of previous address field structure. Update any code that reads or sets billing address data.
- **Checkout and Cart drop-ins**: Event handling now includes a `type` property. Events `checkout/initialized` and `checkout/updated` now include `type: 'cart' | 'quote'`, and Cart only processes events where `type` is `'cart'` (or undefined for backward compatibility). Update event listeners if you filter or process these events.

> - **Order drop-in**: GraphQL mutation changed from `placeNegotiableQuoteOrder` to `placeNegotiableQuoteOrderV2`. Update any custom order placement code that uses this mutation.

Reviewer: ```suggestion
- **Order drop-in**: The `placeNegotiableQuoteOrder` GraphQL mutation has been deprecated in favor of `placeNegotiableQuoteOrderV2`. Update any custom order placement code that uses this mutation.
```

- **User Account drop-in**: GraphQL query structure changed for order items. Fields `product_sku` and `product_url_key` are now at the order item level instead of nested under `product`. Update any code that parses order history data.
- **Boilerplate**: Authentication flow refactored to use page reloads instead of in-place updates. If you have custom authentication integrations, test thoroughly to ensure compatibility with the new flow.

</Aside>

### Highlights

This release introduces several significant enhancements across the Commerce Storefront ecosystem:

> - **Major version 3.0.0 for B2C drop-ins**: All B2C drop-in components have been upgraded to version 3.0.0. These drop-ins have been tested on Adobe Commerce as a Cloud Service (ACCS) and Adobe Commerce 2.4.7+ for Platform-as-a-Service (PaaS) deployments. However, the revised license agreement restricts use to Adobe Commerce as a Cloud Service (ACCS) and Adobe Commerce Optimizer (ACO) only.

Reviewer: ```suggestion
- **Major version 3.0.0 for B2C drop-ins**: All B2C drop-in components have been upgraded to version 3.0.0. These drop-ins have been tested on Adobe Commerce as a Cloud Service (SaaS) and Adobe Commerce 2.4.7+ for Platform-as-a-Service (PaaS) deployments. However, the revised license agreement restricts use to SaaS and Adobe Commerce Optimizer only.
```
